db/migrate/20170324171408_initial_migration.rb

```diff
@@ -8,6 +8,7 @@ def change
 
     create_table :resources do |t|
       t.string :name, null: false
+      t.string :description
```

Comment:
Can you move these changes into their own migration file? Typically you don't want to be editing migrations.